runtime/near-vm-errors/src/lib.rs

```diff
@@ -7,18 +7,24 @@ use serde::{Deserialize, Serialize};
 use std::any::Any;
 use std::fmt::{self, Error, Formatter};
 
-#[derive(Debug, PartialEq, Eq, strum::IntoStaticStr)]
-pub enum VMError {
-    FunctionCallError(FunctionCallError),
-    /// Type erased error from `External` trait implementation.
-    ExternalError(AnyError),
+/// Fatal error occurred during VM operation.
```

Comment:
`Fatal` is a bit unfortunate term here. It *feels* crisp enough to create a feeling of understanding in the reader, but actually explains nothing. 

Could you expand the definition a bit? That is, that this is error originating in runtime itself, rather than the contract, and that  the correct behavior is to either crash or create a challenge? 

The ultimate error handler here I think is 

https://github.com/near/nearcore/blob/0251fca578975beba51df7265af3b6cf7aabc0c6/nearcore/src/runtime/mod.rs#L515-L530